CommunicationLayer/CommDeviceControl/OutputRadioCommDevice.cpp

```diff
@@ -0,0 +1,69 @@
+#include <QDebug>
+#include <QTimer>
+
+#include "OutputRadioCommDevice.h"
+#include "InfrastructureLayer/Settings/I_Settings.h"
+
+OutputRadioCommDevice::OutputRadioCommDevice(I_CommDevice& inputSerialPort, QSerialPort& outputSerialPort, I_Settings& settings)
+    : outputSerialPort_(outputSerialPort)
+    , inputSerialPort_(inputSerialPort)
+{
+    setSerialParameters(settings.outputSerialPortName(), settings.outputBaudrate());
+
+    // This makes sure that it is connected after everything is created.
+    // Only if the config file enables it.
+    if (settings.outputSerialEnabled())
```

Comment:
to not raise erreors, i would just check this at the beginning and then return if false.

Reply:
To make sure you actually do nothing if it's not enabled.

Who guarantees that `setSerialParameters(settings.outputSerialPortName(), settings.outputBaudrate());` doesn't throw any errors? Maybe it doesn't, but whoever is looking at this next will have to read it. Maybe it does, and we could find ourselves with a difficult bug.